common/transform/src/test/java/ai/h2o/mojos/deploy/common/transform/ShapleyLoadOptionTest.java

```diff
@@ -0,0 +1,143 @@
+package ai.h2o.mojos.deploy.common.transform;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import uk.org.webcompere.systemstubs.environment.EnvironmentVariables;
+import uk.org.webcompere.systemstubs.jupiter.SystemStub;
+import uk.org.webcompere.systemstubs.jupiter.SystemStubsExtension;
+
+@ExtendWith(SystemStubsExtension.class)
+public class ShapleyLoadOptionTest {
```

Comment:
nit: No need to make test class public.  Can leave out that modifier and just have `class ...`